### PR TITLE
spelling corrections

### PR DESCRIPTION
**Pull request by Spectre5**
Entire PR is only spelling corrections and there should be no functional change to the code itself and the majority of the corrections are in comments and a few printed strings.

- There are one or two inner function names changed (for example, `get_prinicipal_points` to `get_principal_points`)
- I changed two instances of `’` to `'`
- I left the Aussie "s" spelling on words like discretised (vs the US discretized), ~but I did change artefacts to artifacts as I've never seen it spelled artefacts before~

### CODE REVIEW

File: docs/source/rst/examples.rst [hunk 1]
```diff
@@ -411,7 +411,7 @@ class. The order of the geometry objects in the list that is passed into the con
 order relates to specifying mesh sizes and material properties.
 
 Once the geometry has been merged, it is vital to clean the geometry to remove
-any artefacts that may impede the meshing algorithm. A mesh is created with a mesh
+any artifacts that may impede the meshing algorithm. A mesh is created with a mesh
```
Comment (robbievanleeuwen) [inline docs/source/rst/examples.rst:414]: artefact is British English which is what I use :smile:

File: docs/source/rst/geom_mesh.rst [hunk 2]
```diff
@@ -181,7 +181,7 @@ Cleaning the Geometry
 ---------------------
 
 When creating a merged section often there are overlapping facets or duplicate nodes. These
-geometry artefacts can cause difficulty for the meshing algorithm. It is therefore recommended to
+geometry artifacts can cause difficulty for the meshing algorithm. It is therefore recommended to
```
Comment (robbievanleeuwen) [inline docs/source/rst/geom_mesh.rst:184]: see above.

**Review by robbievanleeuwen — COMMENTED** [review]
Awesome work! Looks like I've made a few copy paste errors (assembing haha). Thanks for picking these up! Only comment is regarding artefact which I believe to be British English.

Thanks!

File: docs/source/rst/examples.rst (hunk 1 above)
Comment (Spectre5) [reply docs/source/rst/examples.rst]: changed back.

File: docs/source/rst/geom_mesh.rst (hunk 2 above)
Comment (Spectre5) [reply docs/source/rst/geom_mesh.rst]: changed back.

**Comment by Spectre5** [comment]
Ok, I changed it back!  I know about "our" endings like colour and "s" vs "z" differences, but artefact is a new one for me :)

**Pull request merged**